File: Seminar5/HomeWork/Model/Complex/SumModelComplex.java
package HomeWork.Model.Complex;

public class SumModelComplex extends CalcModelComplex {

    public SumModelComplex() {

    }

    // do_it
    @Override
    public Complex result() {
        double real1 = x.getReal();
        double image1 = x.getImage();

        double real2 = y.getReal();
        double image2 = y.getImage();

        double newReal = real1 + real2;
        double newImage = image1 + image2;

        Complex result = new Complex(newReal, newImage);

        return result;
    }

    @Override
    public void setX(Complex x) {
        super.x = x;

    }

    @Override
    public void setY(Complex y) {
        super.y = y;

    }

}

File: FinalTaskCalculator/Model/Complex/SumModelComplex.cs
namespace ModelComplex
{
    public class SumModelComplex : CalcModelComplex
    {
        // do_it
        public override Complex Result()
        {
            double real1 = x.Real;
            double image1 = x.Image;

            double real2 = y.Real;
            double image2 = y.Image;

            double newReal = real1 + real2;
            double newImage = image1 + image2;

            Complex result = new Complex(newReal, newImage);

            return result;
        }
        public override void SetX(Complex value) => this.x = value;
        public override void SetY(Complex value) => this.y = value;
    }
}